core/src/main/resources/jenkins/diagnostics/ControllerExecutorsNoAgents/message_pt_BR.properties

```diff
@@ -23,6 +23,5 @@
 Dismiss=Dispensar
 ExecutorsWarning=Constru\u00E7\u00E3o no n\u00F3 embutido pode ser um problema de seguran\u00E7a. \
 Voc\u00EA deve configurar constru\u00E7\u00F5es distribu\u00EDdas. \
-Vide <a href="https://www.jenkins.io/redirect/building-on-controller/" target="_blank" rel="noopener noreferrer">a documenta\u00E7\u00E3o</a>.
```

Comment:
same problem here and in the next few files

Reply:
Indeed. I reviewed and fix all files again.
This looks like a bug to me with `translation_tool.pl`, it is not only removing sentences it shouldn't, it is also messing up with the two spaces prefix when the previous line ends with a `\`.
If you agree, I can open a Jira ticket to address that.